### DISCUSSION TITLE
How to connect self-hosted server via client?

### DISCUSSION THREAD

**Discussion by angryfoxx** [hoppscotch/hoppscotch#4577] [Q&A] [answered]
First of all thanks for the great effort and such a great apps! 
I have Hoppscotch self-hosted server and I want to reach out that server on hoppscotch macOS client. 
Is there any way to connect/reach that server via client.

Hoppscotch version: 24.11.0-0 (20241128.144007)
MacOS version: 15.1.1 (24B91)

**Comment by SanskritiHarmukh** [comment, marked as answer]
Hi there! Currently, the support for connecting the Hoppscotch Desktop app to a self-hosted instance is an active work-in-progress and is a priority feature scheduled for release next year. You can follow the ongoing conversation and updates [here](https://github.com/hoppscotch/hoppscotch/issues/3523).

**Reply by angryfoxx** [reply]
thanks for replying!
